server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -645,6 +645,10 @@
   // time in nanosecond precision when starting up
   private long startUpNanosecond = System.nanoTime();
 
+  // enable new indexer
+  private boolean enableDeviceIndexer = false;
+  private int deviceIndexerType = 0;
```

Comment:
I see that you only add these parameter in config but not in iotdb-engine.properties and IoTDBDescriptor.java. So our user can't see you config, please have a look~

Reply:
At the same time, please don't forget update read me about config change

Reply:
Thank you for you advise!